tests/test_episode.py

```diff
@@ -54,20 +54,22 @@ def test_episode_properties_1(self):
             "Five, Six, Seven, Ate!",
             f"'{episode.episode_name}' was not equal to expected 'Five, Six, Seven, Ate!'",
         )
-        self.assertIsNone(
-            episode.first_aired, "Episode was not expected to have firsted aired data"
+        self.assertIsNotNone(
```

Comment:
These tests are a little out of date and weren't exactly the best (testing an API wrapper is always tough). If this has changed, can it be checked against the exact value please rather than checking it isn't `None`?